frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt

```diff
@@ -118,7 +126,7 @@ class FragNavController constructor(private val fragmentManger: FragmentManager,
             } else {
                 //Else try to find one in the FragmentManager
                 val fragmentManager: FragmentManager = getFragmentManagerForDialog()
-                mCurrentDialogFrag = fragmentManager.fragments?.firstOrNull { it is DialogFragment } as DialogFragment?
+                mCurrentDialogFrag = fragmentManager.fragments.firstOrNull { it is DialogFragment } as DialogFragment?
```

Comment:
The compiler was suggesting to remove safe calls as they're not nullable anymore.